CMakeLists.txt

```diff
@@ -48,21 +48,38 @@ find_package(ZLIB REQUIRED)
 find_package(Threads REQUIRED)
 
 # Clang
-include(cmake/clang/download.cmake)
+if (NOT DOWNLOAD_CLANG)
```

Comment:
If clang LLVM (and the versions we want to support) aren't found - the script defaults to downloading it.

You should you want it to download by default?